src/Swarm/TUI/Model.hs

```diff
@@ -176,26 +179,24 @@ uiInventory :: Lens' UIState (Maybe (Int, BL.List Name InventoryEntry))
 --   error message that is shown on top of the rest of the UI.
 uiError :: Lens' UIState (Maybe (Widget Name))
 
+-- | Computed ticks per milli seconds
+uiTPS :: Lens' UIState Double
```

Comment:
It's not ticks per second that we want to display (the ticks per second is set directly by the user), but rather the ticks per frame.

Reply:
Oops, I see :)